doc/onboarding.md

```diff
@@ -0,0 +1,88 @@
+# Numberscope onboarding
+
+## Accounts, etc.
+
+1. If you don't have an account, sign up for [GitHub](https://github.com).
+2. Make sure you've been added to the Numberscope GitHub organization.
+3. Make sure you've been added to the chat app (Slack or Zulip).
+
+## Setting up your computer for development
+
+1. If you don't have one, install a good text editor. Use whatever you want!
+   [Visual Studio Code](https://code.visualstudio.com/) is popular, and it
+   works well with our code base. If you want to use Visual Studio Code,
+   install the following extensions:
+
+    - [ ] Code Spell Checker
+    - [ ] TypeScript Vue Plugin (Volar)
+    - [ ] Vue Language Features (Volar)
+
+    For more info on extensions, see:
+    https://code.visualstudio.com/docs/editor/extension-marketplace.
+
+2. If you don't have [Git](https://git-scm.com/) installed, install it. We use
+   Git to keep track of the different versions of our code.
+3. If you don't have [NodeJS](https://nodejs.org/en/) installed, install it.
+   NodeJS allows you to run JavaScript outside of a web browser. We use it in
+   our front end code base.
+4. If you don't have [Python 3](https://www.python.org/) installed, install
+   it.
+5. Make sure you have a `venv` module by running the following command:
+    ```sh
+    % python3 -m venv -h
+    ```
+    You should see help for the `venv` module. If you don't you might have to
+    install the module separately. How you do this depends on your system and
+    might require some googling.
+6. Somewhere on your computer, make a directory where you can keep Numberscope
+   code. I like to put a directory called `Code` in my home directory. My home
+   directory looks like this:
+    ```
+    ├── Applications
+    ├── Books
+    ├── Code
+    ├── Desktop
+    ├── Documents
+    ├── Downloads
+    ├── Library
+    ├── Movies
+    ├── Music
+    ├── Pictures
+    └── Public
+    ```
+    My `Code` directory looks like this:
```

Comment:
Are Music, Pictures, Movies, etc. TMI for this? Maybe stop after Downloads and put "..."? Also, I think this is a Mac-ish home directory organization, if I am not mistaken. Will it be confusing to PC people? Could you even perhaps get away with not really showing your home directory, just mentioning that you put a folder called Code in there?

Reply:
Got rid of this in https://github.com/numberscope/frontscope/pull/148/commits/59fd25623bc1f78bae3e0e097eb8fbecb53e6926.